admin/framework/phpunit.xml.dist

```diff
@@ -1,60 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="system/Test/bootstrap.php"
-		backupGlobals="false"
-		colors="true"
-		convertErrorsToExceptions="true"
-		convertNoticesToExceptions="true"
-		convertWarningsToExceptions="true"
-		stopOnError="false"
-		stopOnFailure="false"
-		stopOnIncomplete="false"
-		stopOnSkipped="false">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="system/Test/bootstrap.php" backupGlobals="false" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" stopOnError="false" stopOnFailure="false" stopOnIncomplete="false" stopOnSkipped="false" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
Can we make this as multiline for better readability?

Reply:
This is the output from PHPUnit's `--migrate-configuration`. I already converted it to tabs so I'm fine modifying it more - just noting where the differences come from.